books/free-programming-books-id.md

```diff
@@ -158,10 +158,13 @@
 ### Python
 
 * [Belajar Python](http://www.belajarpython.com)
+* [Belajar python bahasa Indonesia](https://sekolahkoding.com/track/belajar-python-bahasa-indonesia) - Sekolah Koding
```

Comment:
this is mostly video content - doesn't it fit better in the courses list?

Reply:
ah yes you're right, it's more suitable to put in course list. thankyou for the correction!